android/src/main/java/com/klaviyoreactnativesdk/KlaviyoReactNativeSdkModule.kt

```diff
@@ -60,16 +61,28 @@ class KlaviyoReactNativeSdkModule(
   }
 
   @ReactMethod
-  fun registerForInAppForms() {
+  fun registerForInAppForms(configuration: ReadableMap?) {
     UiThreadUtil.runOnUiThread {
       try {
-        Klaviyo.registerForInAppForms()
+        val sessionTimeoutDuration = configuration?.getDouble("sessionTimeoutDuration")
+        if (sessionTimeoutDuration != null) {
+          Klaviyo.registerForInAppForms(com.klaviyo.forms.IAFConfiguration(sessionTimeoutDuration = sessionTimeoutDuration))
```

Comment:
Late now, but couldn't we import the configuration data class so it doesn't have to be fully qualified here